### PR TITLE
Add chi middleware

### PR DESCRIPTION
**Pull request by milarisa**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -36,6 +36,7 @@ require (
 	github.com/fatih/structs v1.1.0 // indirect
 	github.com/flosch/pongo2/v4 v4.0.2 // indirect
 	github.com/gin-contrib/sse v0.1.0 // indirect
+	github.com/go-chi/chi v1.5.4 // indirect
```
Comment (sashayakovtseva) [inline go.mod]: pls double check this cannot be removed

File: middleware/chi/example_test.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	metrics "github.com/slok/go-http-metrics/metrics/prometheus"
+	"github.com/slok/go-http-metrics/middleware"
+	chiMiddleware "github.com/slok/go-http-metrics/middleware/chi"
+)
+
+// ChiMiddleware shows how you would create a default middleware
+// factory and use it to create a chi compatible middleware.
+func Example_chiMiddleware() {
```
Comment (sashayakovtseva) [inline middleware/chi/example_test.go:15]: ExampleChiMiddleware?

Comment (milarisa) [reply middleware/chi/example_test.go:15]: made in the same style as everywhere: `Example_fasthttpMiddleware` `Example_stdMiddleware`

File: go.mod (hunk 1 above)
Comment (milarisa) [reply go.mod]: you're right, fixed

**Review by sashayakovtseva — APPROVED** [review]

**Pull request merged**